### DISCUSSION TITLE
Custom fonts

### DISCUSSION THREAD

**Discussion by 97-42** [vkbo/novelWriter#1682] [Ideas]
I would like to use custom font that can be uploaded by the user's computer. I think this will open a whole new windows for this project.

**Comment by vkbo** [comment]
Not sure what you mean by "uploaded font"? novelWriter can use any font installed on your system.

**Reply by 97-42** [reply]
can you please tell me how?
BTW i have read the manual and cannot find any relevant information and that was a very quick reply, cheers

**Reply by vkbo** [reply]
In Preferences you can select font for both the user interface and the documents. They are separate font selections because you may want to use different fonts.

**Reply by 97-42** [reply]
Will it work with .tff files?

**Reply by vkbo** [reply]
Yes, it should. Just install the fonts you want to use on your system. The font selector in Preferences just uses your system's native font selector and loads it. There is no custom implementation here. It's all standard functionality of the framework novelWriter is built on.

**Reply by vkbo** [reply]
I see you deleted the last comment. I assume you found the document font setting?
